tests/test_schema.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2018 Leiden University Medical Center
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/
+
+# This file contains tests for the schema for the yaml file
+
+from pytest_workflow.schema import validate_schema
+import pytest
+import yaml
+from pathlib import Path
+
+def valid_yamls():
```

Comment:
Why is this a function and not simply a list?  It returns the same thing every time it's invoked, so it can be a constant.